<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Copyright 2010, Martin Koller</author>
 <class>PictureButtonSettings</class>
 <widget class="QDialog" name="PictureButtonSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bilder Einstellungen</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Number of images</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QSpinBox" name="numButtons">
       <property name="maximum">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QListWidget" name="listWidget"/>
     </item>
     <item row="2" column="1">
      <widget class="QPlainTextEdit" name="textEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
       <property name="icon">
        <iconset resource="LillePOS.qrc">
         <normaloff>:/icons/ok.png</normaloff>:/icons/ok.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="LillePOS.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>PictureButtonSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>717</x>
     <y>456</y>
    </hint>
    <hint type="destinationlabel">
     <x>485</x>
     <y>457</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
